lib/qunit-bdd.js

```diff
@@ -26,6 +26,28 @@ void function(global, exports, QUnit, options) {
     this.lazyTeardown = [];
   }
 
+  function wrapHookCallback(fn, hook) {
+    // QUnit calls hooks with `assert` as the param and the `testEnvironment` as the `this` context
+    return function(assert) {
+      if (QUnit.config.notrycatch) {
+        return fn.call(this, assert);
+      } else {
+        try {
+          return fn.call(this, assert);
+        } catch (e) {
+          if (e.stack) {
+            console.error(e.stack);
+          }
+          QUnit.pushFailure(
+            'Exception while running qunit-bdd `' + hook + '` hook: ' +
```

Comment:
we're duplicating QUnit's `notrycatch` if/else in order to provide a bdd-tailored exception message. We return the function without any of this, but error messages from a qunit-bdd `before` would show up as if they were in a qunit `beforeEach` which could cause confusion.